src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -42,7 +42,7 @@ type ClusterStatus struct {
 
 	// Replicas show how many nodes are working in the cluster
 	// +optional
```

Comment:
Should we :scissors: `// +optional` too to be fully aligned with upstream `StatefulSetStatus` ?
I wonder if it breaks anything on our end.

Reply:
oh dang sorry. abou this merge. @alenkacz will need to follow up on a new PR with this suggestion.